hibernate-core/src/main/java/org/hibernate/cfg/AvailableSettings.java

```diff
@@ -1511,4 +1511,15 @@
 	 */
 	String ALLOW_JTA_TRANSACTION_ACCESS = "hibernate.jta.allowTransactionAccess";
 
+	/**
+	 *  Setting that allows to perform update operations outside a transaction boundary.
```

Comment:
I think "outside of a transaction boundary" would read better than "outside a transaction boundary"